assets/js/lunr-search.js

```diff
@@ -59,13 +59,21 @@
 
     function searchDone() {
         form.removeAttribute("data-running");
+
+        // A magic trick to make search field loses focus on mobile,
+        // which prevents the virtual keyboard from popping up.
+        const header = document.querySelector('.header');
+        if (header && header.className !== 'header') {
```

Comment:
`header && header.classList.contains("open")` I guess? This shouldn't be triggered by random classes that the site added to the header.

Reply:
I don't know why, but `header.classList.contains("hide")` works when I tested with a real device, kinda weird...